modules/data.land/R/shp2gpkg.R

```diff
@@ -0,0 +1,84 @@
+#' Convert Shapefile to GeoPackage
+#'
+#' This function converts a Shapefile to a GeoPackage using the `sf` package.
+#'
+#' @param input_shp Character. Path to the input Shapefile (e.g., `"data/myfile.shp"`).
+#' @param output_gpkg Character. Path to the output GeoPackage (e.g., `"output/myfile.gpkg"`).
+#' @param layer_name Character. Optional. Name of the layer in the GeoPackage. Defaults to the base name of the Shapefile.
+#' @param overwrite Logical. Whether to overwrite an existing layer in the GeoPackage. Defaults to `TRUE`.
+#'
+#' @return Invisibly returns the path to the GeoPackage (`output_gpkg`) upon successful conversion.
+#' @details
+#' This function reads a Shapefile, converts it to an `sf` object, and writes it to a GeoPackage.
+#' The `sf` package handles the conversion and ensures spatial data integrity.
```

Comment:
Nit: I recommend moving details above the parameter listing and removing the @details tag -- that's the layout Roxygen favors and it treats paragraph 1 as the title, paragraph 2 as the description, and paragraphs 3-and-onward as details. Yes, it differs from the layout of an Rd page, but I find it makes it easier to find the parameters when reading in source form.